v1.16/QKE/README.md

```diff
@@ -0,0 +1,42 @@
+## How to Reproduce
+
+### Login 
+
+Login to [QingCloud Console Platform](https://console.qingcloud.com/) Website with your own qingcloud account.
+
+### Create QKE Cluster
+
+Navigate to QKE cluster, Create a ap2a Region(important) QKE cluster according to [QKE Guide](https://docs.qingcloud.com/product/container/qke/index).
+
+![](cluster.png)
+
+>> Note:
+>> pls select ap2a region
```

Comment:
please choose ap2a or ap3 region to avoid network issues when pulling gcr.io images

Reply:
ok